liblangutil/Token.h

```diff
@@ -316,6 +319,13 @@ namespace TokenTraits
 	constexpr bool isTimeSubdenomination(Token op) { return op == Token::SubSecond || op == Token::SubMinute || op == Token::SubHour || op == Token::SubDay || op == Token::SubWeek || op == Token::SubYear; }
 	constexpr bool isReservedKeyword(Token op) { return (Token::After <= op && op <= Token::Unchecked); }
 
+	constexpr bool isYulToken(Token tok)
```

Comment:
What is a "yul token"? Should this say `isYulKeyword`?

Reply:
Fine with me.

Reply:
Fine with either.

Reply:
Changed to `isYulKeyword`.